addons/mail/models/discuss/discuss_channel_member.py

```diff
@@ -318,7 +324,7 @@ def _rtc_join_call(self, store: Store = None, check_rtc_session_ids=None, camera
                     "serverInfo": self._get_rtc_server_info(rtc_session, ice_servers),
                 },
             )
-        if len(self.channel_id.rtc_session_ids) == 1:
+        if len(self.channel_id.rtc_session_ids) == 1 and self._should_invite_members_to_join_call():
```

Comment:
If `len(self.channel_id.rtc_session_ids) == 1` is added in `_should_invite_members_to_join_call` it doesn't need to be repeated here.